1_crosswalk_fetch/src/get_ned_points.R

```diff
@@ -0,0 +1,26 @@
+
+#' get point elevation data from the `elevatr` package
+#'
+#' @param point_rds spatial data file for points
+#' @param zoom zoom level, input into `elevatr::get_aws_points` function
+#' @param dummy character that isn't used but if changed, will cause rebuilds
+#'
+#' @details
+#' needed to install CRAN binary of sf and downgrade PROJ/GDAL to get past error:
```

Comment:
Yes, this would prevent someone from rebuilding. But I was the only one apparently running into this issue and it was because I was on the very edge development version of `sf`, `GDAL` and `PROJ`, so others were not seeing it. The package developer has added the more recent dev edge package on their mac tests, so they are aware of the issue and will probably resolve it. But I'd bet you can run this code right out of the box right now if you have these packages installed.